mods/d2k/tilesets/arrakis.yaml

```diff
@@ -4240,140 +4240,140 @@ Templates:
 		Size: 1,1
 		Category: Sand-Detail
 		Tiles:
-			0: Cliff
+			0: Rough
 	Template@501:
 		Id: 501
 		Images: BLOXXMAS.R8
 		Frames: 325
 		Size: 1,1
 		Category: Sand-Detail
 		Tiles:
-			0: Cliff
+			0: Sand
 	Template@502:
 		Id: 502
 		Images: BLOXXMAS.R8
 		Frames: 311
 		Size: 1,1
 		Category: Sand-Detail
 		Tiles:
-			0: Cliff
+			0: Rough
 	Template@503:
 		Id: 503
 		Images: BLOXXMAS.R8
 		Frames: 312
 		Size: 1,1
 		Category: Sand-Detail
 		Tiles:
-			0: Cliff
+			0: Rough
 	Template@504:
 		Id: 504
 		Images: BLOXXMAS.R8
 		Frames: 330
 		Size: 1,1
 		Category: Sand-Detail
 		Tiles:
-			0: Cliff
+			0: Rough
 	Template@505:
 		Id: 505
 		Images: BLOXXMAS.R8
 		Frames: 316
 		Size: 1,1
-		Category: Sand-Detail
+		Category: Bridge
 		Tiles:
-			0: Cliff
+			0: Rough
 	Template@506:
 		Id: 506
 		Images: BLOXXMAS.R8
 		Frames: 297
 		Size: 1,1
-		Category: Sand-Detail
+		Category: Bridge
 		Tiles:
-			0: Cliff
+			0: Rough
 	Template@507:
 		Id: 507
 		Images: BLOXXMAS.R8
 		Frames: 356, 357
 		Size: 2,1
 		Category: Sand-Detail
 		Tiles:
-			0: Cliff
-			1: Cliff
+			0: Rough
+			1: Rough
 	Template@508:
 		Id: 508
 		Images: BLOXXMAS.R8
 		Frames: 355
 		Size: 1,1
 		Category: Sand-Detail
 		Tiles:
-			0: Cliff
+			0: Rough
 	Template@509:
 		Id: 509
 		Images: BLOXXMAS.R8
 		Frames: 375
 		Size: 1,1
 		Category: Sand-Detail
 		Tiles:
-			0: Cliff
+			0: Rough
 	Template@510:
 		Id: 510
 		Images: BLOXXMAS.R8
 		Frames: 395
 		Size: 1,1
 		Category: Sand-Detail
 		Tiles:
-			0: Cliff
+			0: Rough
 	Template@511:
 		Id: 511
 		Images: BLOXXMAS.R8
 		Frames: 414, 415, 434, 435
 		Size: 2,2
 		Category: Sand-Detail
 		Tiles:
-			0: Cliff
-			1: Cliff
-			2: Cliff
-			3: Cliff
+			0: Rough
+			1: Rough
+			2: Rough
+			3: Rough
 	Template@512:
 		Id: 512
 		Images: BLOXXMAS.R8
 		Frames: 353, 354, 373, 374, 393, 394
 		Size: 2,3
-		Category: Sand-Detail
+		Category: Bridge
 		Tiles:
-			0: Cliff
-			1: Cliff
-			2: Cliff
-			3: Cliff
-			4: Cliff
-			5: Cliff
+			0: Rough
+			1: Rough
+			2: Rough
+			3: Rough
+			4: Rough
+			5: Rough
 	Template@513:
 		Id: 513
 		Images: BLOXXMAS.R8
 		Frames: 350, 351, 352, 370, 371, 372, 390, 391, 392
 		Size: 3,3
-		Category: Rock-Detail
+		Category: Bridge
 		Tiles:
 			0: Rock
 			1: Rock
 			2: Rock
 			3: Cliff
```

Comment:
`Tile 0` and `Tile 2` should be changed to `Cliff` 

![image](https://cloud.githubusercontent.com/assets/20945684/26363722/2dba7c40-3fe2-11e7-92d4-b816f39ff166.png)